src/apiCalls.js

```diff
@@ -1,14 +1,16 @@
 const getCocktails = (text) => {
-  return fetch(`https://www.thecocktaildb.com/api/json/v1/1/search.php?s=${text}&api_key='1'`)
-  
-    .then(response => {
-      // console.log(response);
+  return fetch(
+    `https://www.thecocktaildb.com/api/json/v1/1/search.php?s=${text}&api_key='1'`
+  )
+    .then((response) => {
+      if (response.status === 404) {
+        throw new Error('404');
+      }
       if (!response.ok) {
-        throw new Error(response.statusText); 
+        throw new Error(response.statusText);
```

Comment:
Is there anyway to consolidate these Error catches so there arent 2 if statements here?